neqo-transport/src/connection.rs

```diff
@@ -4791,4 +4793,21 @@ mod tests {
         assert_ne!(delay, Duration::from_secs(0));
         assert!(delay > lr_time);
     }
+
+    #[test]
+    fn unknow_version() {
+        let mut client = default_client();
+        // Start the handshake.
+        let _ = client.process(None, now()).dgram();
+
+        const UNKNOW_VERSION_PACKET: &[u8] = &[
```

Comment:
```suggestion
        const UNKNOWN_VERSION_PACKET: &[u8] = &[
```

This could be smaller.  But it might be better to pad to 1200 so that we hit the full "initial" path.  For that, you might be better using `Vec::resize`